Incidents support for Cloud Security Platform (#14014)
DataDog/documentation: content/en/monitors/incident_management/_index.md

{{< img src="monitors/incidents/existing.png" alt="Add a monitor to an existing incident" style="width:80%;">}}

#### From a Security Signal

> Declare an incident directly from a Cloud SIEM or Cloud Workload Security signal by clicking the kebab button in the top right of the side panel, or by selecting the export button in the Application Security Monitoring side panel, and clicking **Declare incident**. The incident creation modal appears, and the security signal is added to the incident as a signal.

Reviewer: ```suggestion
Declare an incident directly from a Cloud SIEM or Cloud Workload Security signal by clicking the kebab button on the top right of the side panel, and clicking **Declare incident**.

Declare an incident from an Application Security Monitoring signal by selecting the export button on the top right of the side panel, and clicking **Export to incident**.
```

Author: lgtm, ty!

Reviewer: Thanks @amberben ! Just a note: before resolving, if you the changes look good, you'll need to "Commit suggestion" or, if there are multiple suggestions, "Add suggestion to batch" and then batch commit them all. Committing the suggestions pushes the changes through. I just added the changes again and committed them, so we are set here. :)
